### PR TITLE
Snowflake integration test steps

### PR DESCRIPTION
**Pull request by bleonard**
Adding notes on how to set up the test environment

### CODE REVIEW

File: airbyte-integrations/connectors/destination-snowflake/README.md [hunk 1]
```diff
@@ -8,23 +8,76 @@
...
 
 1. Put the contents of the `destination snowflake - insert test creds` LastPass secret into `secrets/insert_config.json`.
-1. Put the contents of the `destination snowflake - insert staging test creds` secret into `internal_staging_config.json`.
-1. Put the contents of the `destination snowflake - gcs copy test creds` secret into `secrets/copy_gcs_config.json`
-1. Put the contents of the `destination snowflake - s3 copy test creds` secret into `secrets/copy_s3_config.json`
-1. Put the contents of the `destination snowflake - azure blob copy test creds` secret into `secrets/copy_azure_blob_config.json`
+2. Put the contents of the `destination snowflake - insert staging test creds` secret into `internal_staging_config.json`.
+3. Put the contents of the `destination snowflake - gcs copy test creds` secret into `secrets/copy_gcs_config.json`
+4. Put the contents of the `destination snowflake - s3 copy test creds` secret into `secrets/copy_s3_config.json`
+1. Put the contents of the `destination snowflake - s3 copy encrypted test creds` secret into `secrets/copy_s3_encrypted_config.json`
```
Comment (edgao) [inline airbyte-integrations/connectors/destination-snowflake/README.md:30]:
```suggestion
5. Put the contents of the `destination snowflake - s3 copy encrypted test creds` secret into `secrets/copy_s3_encrypted_config.json`
```

File: airbyte-integrations/connectors/destination-snowflake/README.md [hunk 2]
```diff
@@ -8,23 +8,76 @@
...
+GRANT ALL PRIVILEGES ON WAREHOUSE INTEGRATION_TEST_WAREHOUSE_DESTINATION TO ROLE INTEGRATION_TESTER_DESTINATION;
+GRANT ALL PRIVILEGES ON DATABASE INTEGRATION_TEST_DESTINATION TO ROLE INTEGRATION_TESTER_DESTINATION;
+
+GRANT ALL PRIVILEGES ON FUTURE SCHEMAS IN DATABASE INTEGRATION_TEST_DESTINATION TO ROLE INTEGRATION_TESTER_DESTINATION;
+GRANT ALL PRIVILEGES ON FUTURE TABLES IN DATABASE INTEGRATION_TEST_DESTINATION TO ROLE INTEGRATION_TESTER_DESTINATION;
+
+GRANT USAGE ON INTEGRATION GCS_AIRBYTE_INTEGRATION TO ROLE INTEGRATION_TESTER_DESTINATION;
+GRANT USAGE ON STAGE GCS_AIRBYTE_STAGE TO ROLE INTEGRATION_TESTER_DESTINATION;
+
+CREATE USER INTEGRATION_TEST_USER_DESTINATION PASSWORD='test' DEFAULT_ROLE=INTEGRATION_TESTER_DESTINATION DEFAULT_WAREHOUSE=INTEGRATION_TEST_WAREHOUSE_DESTINATION MUST_CHANGE_PASSWORD=false;
```
Comment (edgao) [inline airbyte-integrations/connectors/destination-snowflake/README.md:66]: nitpick (making it easier to find the password to modify)
```suggestion
# Modify this password
CREATE USER INTEGRATION_TEST_USER_DESTINATION PASSWORD='test' DEFAULT_ROLE=INTEGRATION_TESTER_DESTINATION DEFAULT_WAREHOUSE=INTEGRATION_TEST_WAREHOUSE_DESTINATION MUST_CHANGE_PASSWORD=false;
```

File: airbyte-integrations/connectors/destination-snowflake/README.md [hunk 3]
```diff
@@ -8,23 +8,76 @@
...
+  "credentials": {
+    "password": "test"
+  }
 }
 ```
 
 ## For Airbyte employees
 Put the contents of the `Snowflake Integration Test Config` secret on LastPass under the `Engineering` folder into `secrets/config.json` to be able to run integration tests locally.
 
 1. Put the contents of the `destination snowflake - insert test creds` LastPass secret into `secrets/insert_config.json`.
```
Comment (edgao) [inline airbyte-integrations/connectors/destination-snowflake/README.md:26]: the lastpass creds are still using the old values - are you planning to update them, or should this list be modified to reference the GSM entries?

(hackweek thought: can fetching these secrets be done via script)

**Review by edgao — APPROVED** [review]
lgtm pending a few minor comments

File: airbyte-integrations/connectors/destination-snowflake/README.md (hunk 3 above)
Comment (bleonard) [reply airbyte-integrations/connectors/destination-snowflake/README.md:26]: I'll do a future update here for dev credentials. One prong of the current work is to separate out the CI world fro the dev world.

Comment (bleonard) [reply airbyte-integrations/connectors/destination-snowflake/README.md:26]: hack day delivered?! https://github.com/airbytehq/airbyte/blob/master/tools/ci_credentials/ci_credentials/secrets_loader.py

**Comment by sherifnada** [comment]
test comment

**Review by evantahler — APPROVED** [review]

File: airbyte-integrations/connectors/destination-snowflake/README.md (hunk 3 above)
Comment (edgao) [reply airbyte-integrations/connectors/destination-snowflake/README.md]: whoa, how did I not hear about this?

do we not want to keep CI and dev secrets identical? (might be easier to move this discussion out of github comments tbh)

**Pull request merged**